setup.py

```diff
@@ -27,6 +27,7 @@
         'urllib3<2.0',
         'packageurl-python==0.10.3',
         'GitPython==3.1.29',
+        'rpm',
```

Comment:
Can you please provide a fixed version?

Reply:
I've updated my PR on commit https://github.com/AlmaLinux/alma-sbom/pull/40/commits/856a4c34a22e0537b3864b0755231611fa9457b3

The rpm module version is difference between venv Enviroment or not, So I've added enviroment check logic.

Note that, I couldn't properly specify versions using the standard notation.
Reference: https://peps.python.org/pep-0508/

Reply:
Oh, I see, when we are in a virtualenv, we need to install the `rpm-shim` module in addition to the system package that provides the `rpm` module, which should be installed anyway.  Can you also please describe this in the README.md file inside the requirements section?